src/test/testFeeds.ts

```diff
@@ -9,73 +9,27 @@ import {
   randImage,
 } from "./testhelpers";
 import {
-  createImage,
-  createVideo,
   ActivityContentNote,
   ActivityContentAttachment,
-  ActivityContentPerson,
-  ActivityContentImage,
-  ActivityContentVideo,
+  ActivityContentProfile,
 } from "@dsnp/sdk/core/activityContent";
 
-/**
- * Generate an image attachment from a given url.
- * @param url The url to generate the attachment around
- */
-export const generateImageAttachment = (url: string): ActivityContentImage => {
-  return createImage(
-    url,
-    url.replace(/(^\w+:|^)\/\//, ""),
-    { algorithm: "" },
-    0,
-    0
-  );
-};
-
-/**
- * Generate a video attachment from a given url.
- * @param url The url to generate the attachment around
- */
-export const generateVideoAttachment = (url: string): ActivityContentVideo => {
-  return createVideo(
-    url,
-    url.replace(/(^\w+:|^)\/\//, ""),
-    { algorithm: "" },
-    0,
-    0
-  );
-};
-
-/**
- * Generate a Note piece of Content for us in constructing a Feed
- * @param address the HexString socialAddress to associate with making this note
- * @param message The message string to display in the note
- * @param attachment the NoteAttachments for pictures and videos in this Note.
- */
-export const generateNote = (
-  message: string,
-  attachment?: ActivityContentAttachment[]
-): ActivityContentNote => {
-  return {
-    "@context": "https://www.w3.org/ns/activitystreams",
-    content: message,
-    attachment: attachment || [],
-    type: "Note",
-    published: "17ad9cb8551",
-  };
-};
+import { generators } from "@dsnp/sdk";
+const generateImageAttachment =
+  generators.activityContent.generateImageAttachment;
+const generateNote = generators.activityContent.generateNote;
 
 /**
  * Generate a Profile update. The type `Person` is not a `Profile`
  * @param address the HexString socialAddress to make this person update around
  * @param name the new name of the profile update
  * @param handle the new username of the profile update
  */
-export const generatePerson = (name?: string): ActivityContentPerson => {
+export const generatePerson = (name?: string): ActivityContentProfile => {
   return {
     "@context": "https://www.w3.org/ns/activitystreams",
     name: name || "",
-    type: "Person",
+    type: "Profile",
     published: "17ad9cb8551",
```

Comment:
Non-blocking: `published` isn't actually required on the ActivityContent anymore now that we added it to the announcements.